Makefile

```diff
@@ -0,0 +1,17 @@
+all: generate_templates install
```

Comment:
I don't think we need this Makefile, as we can run the local tool without installing, explicitly. A go generate
statement would look like `//go:generate go run github.com/deepmap/oapi-codegen/cmd/oapi-codegen  ...`

Reply:
OK I removed the makefile (it was temporarily useful for me because I was building/generating from my fork which was checked out into a different dir)